k9mail-library/src/main/java/com/fsck/k9/mail/Flag.java

```diff
@@ -1,53 +1,84 @@
-
 package com.fsck.k9.mail;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+
+import timber.log.Timber;
+
+
 /**
  * Flags that can be applied to Messages.
+ *
+ * Terminology:
```

Comment:
I'd rather see this go on the website's developer documentation than in the source, but I'm a clean-code-ist.

Reply:
Here we go: See [#55](https://github.com/k9mail/k9mail.github.io/pull/55) for k9mail.github.io.